lang/python/python-package.mk

```diff
@@ -35,6 +35,10 @@ ifdef CONFIG_USE_MIPS16
   TARGET_CFLAGS += -mno-mips16 -mno-interlink-mips16
 endif
 
+ifeq ($(ARCH),aarch64)
+  TARGET_CFLAGS += -fPIC
```

Comment:
curio: shouldn't this be set on all ARCHs?
and if yes, you could probably do   `TARGET_CFLAGS += $(FPIC)`